src/HeatPump.cpp

```diff
@@ -235,6 +235,21 @@ heatpumpSettings HeatPump::getSettings() {
   return currentSettings;
 }
 
+heatpumpSettings HeatPump::getWantedSettings()
+{
+  return wantedSettings;
+}
+
+unsigned long HeatPump::getLastWanted()
```

Comment:
Where is this used?

Reply:
You can check it here in my project: https://github.com/dzungpv/mitsubishi2MQTT/blob/9c2bcdda142fff1290fbef9550f4f9214b3f510f/main/main.cpp#L2051. Code have been run on esp8266 to 60 days without any reboot.